src/objects.js

```diff
@@ -1,41 +1,49 @@
 const createPerson = (name, age) => {
-  // your code here
+  return {name, age};
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  if (object[property]) return true;
+  return false;
 };
 
 const isOver65 = person => {
-  // your code here
+  if (person.age > 65) return true;
```

Comment:
You don't need the if statement here. `person.age > 65` will evaluate to a boolean for you.